### PR TITLE
Maxbounds on Map props

### PR DESCRIPTION
**Pull request by SBRK**

### CODE REVIEW

File: src/map.tsx [hunk 1]
```diff
@@ -264,7 +265,7 @@ const ReactMapboxFactory = ({
         zoom: zoom ? zoom[0] : defaultZoom[0],
         minZoom,
         maxZoom,
-        maxBounds,
+        maxBounds: this.props.maxBounds || maxBounds,
```
Comment (alex3165) [inline src/map.tsx]: Can you remove ` || maxBounds` and remove it from the factory parameter (line 197), also destruct `maxBounds` from `this.props` (line 25*)

**Comment by alex3165** [comment]
Nice job, I left a comment, could you update the documentation accordingly?

**Comment by SBRK** [comment]
@alex3165 I updated the doc and removed maxBounds from the factory, but the tests are failing now and I have no idea why (because I see `maxBounds` nowhere on the tests, so I don't know what's going on)

**Comment by alex3165** [comment]
Look like tests are passing on the CI, I will merge the PR and check on master. Nice job 👍

**Comment by SBRK** [comment]
Yeah, I forgot to `npm i` before building :)

**Comment by alex3165** [comment]
Released with version `2.4.0`

**Pull request merged**